docs/iov-name-service/validator/04-upgrade.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
This step is too vital to not be included in the Setup page.  It's a huge pain for validators and us to deal with a new pub_key on every upgrade.